docs/source/testing/larva/options.rst

```diff
@@ -0,0 +1,265 @@
+Options
+========
+
+When creating Larva tests, you will find that tests always fail at certain points because there is a difference between the current result of the test and the stored result of a test, for example, the “timestamp”. After all, time changes every second. For solving this kind of issue, Larva has several options for ignoring /replacing parts of messages to compare them. This subsection is intended to name and describe all ignore, replace and other options within the Larva testing tool.
+
+.. list-table:: Overview
+   :widths: 30 70
```

Comment:
I cloned your pull request and built the HTML. Then I saw that the text does not fit on the screen. Can you fix that?

Reply:
I checked on Friday April 23. This issue has not been solved.

Reply:
It still does not fit on my laptop. Probably it does on yours, Bibi, because that was the case when we discussed this issue before. I think it is best when I deal with this myself.